commit 0719638dfdc3bf8f1d58ee6cf29e591391bc41c8
Author: Romaric Jodin <rjodin@google.com>
Date:   Tue Oct 14 17:07:59 2025 +0200

    meson: update xml files list in mapi
    
    Some XML files are missing when trying to compile within Android
    build system (using ninja-to-soong to convert meson to blueprint).
    
    Sorted list extracted with:
    `find src/mapi/glapi/gen -name *.xml | sed
    "s|src/mapi/\(.*\)|  \'\1\',|" | sort`
    
    Ref #14072
    
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/37864>

diff --git a/src/mapi/meson.build b/src/mapi/meson.build
index 1f32eaea5c6..7a22194beb8 100644
--- a/src/mapi/meson.build
+++ b/src/mapi/meson.build
@@ -6,15 +6,13 @@ glapi_xml_py_deps = files(
   # The purpose of this list is to trigger rebuilds when these files are changed.
   #
   # XML files are first (python files are at the end)
-  'glapi/gen/gl_API.dtd',
-  'glapi/gen/es_EXT.xml',
-  'glapi/gen/gl_API.xml',
-  'glapi/gen/gl_and_es_API.xml',
-  'glapi/gen/gl_and_glX_API.xml',
-  'glapi/gen/glX_API.xml',
+  'glapi/gen/AMD_depth_clamp_separate.xml',
+  'glapi/gen/AMD_draw_buffers_blend.xml',
+  'glapi/gen/AMD_gpu_shader_int64.xml',
+  'glapi/gen/AMD_performance_monitor.xml',
   'glapi/gen/ARB_base_instance.xml',
-  'glapi/gen/ARB_blend_func_extended.xml',
   'glapi/gen/ARB_bindless_texture.xml',
+  'glapi/gen/ARB_blend_func_extended.xml',
   'glapi/gen/ARB_clear_buffer_object.xml',
   'glapi/gen/ARB_clear_texture.xml',
   'glapi/gen/ARB_clip_control.xml',
@@ -28,8 +26,8 @@ glapi_xml_py_deps = files(
   'glapi/gen/ARB_depth_buffer_float.xml',
   'glapi/gen/ARB_depth_clamp.xml',
   'glapi/gen/ARB_direct_state_access.xml',
-  'glapi/gen/ARB_draw_buffers.xml',
   'glapi/gen/ARB_draw_buffers_blend.xml',
+  'glapi/gen/ARB_draw_buffers.xml',
   'glapi/gen/ARB_draw_elements_base_vertex.xml',
   'glapi/gen/ARB_draw_indirect.xml',
   'glapi/gen/ARB_draw_instanced.xml',
@@ -40,27 +38,28 @@ glapi_xml_py_deps = files(
   'glapi/gen/ARB_get_program_binary.xml',
   'glapi/gen/ARB_get_texture_sub_image.xml',
   'glapi/gen/ARB_gl_spirv.xml',
+  'glapi/gen/ARB_gpu_shader5.xml',
   'glapi/gen/ARB_gpu_shader_fp64.xml',
   'glapi/gen/ARB_gpu_shader_int64.xml',
-  'glapi/gen/ARB_gpu_shader5.xml',
   'glapi/gen/ARB_indirect_parameters.xml',
   'glapi/gen/ARB_instanced_arrays.xml',
-  'glapi/gen/ARB_internalformat_query.xml',
   'glapi/gen/ARB_internalformat_query2.xml',
+  'glapi/gen/ARB_internalformat_query.xml',
   'glapi/gen/ARB_invalidate_subdata.xml',
   'glapi/gen/ARB_map_buffer_range.xml',
   'glapi/gen/ARB_multi_bind.xml',
   'glapi/gen/ARB_pipeline_statistics_query.xml',
   'glapi/gen/ARB_program_interface_query.xml',
   'glapi/gen/ARB_robustness.xml',
-  'glapi/gen/ARB_sample_shading.xml',
   'glapi/gen/ARB_sampler_objects.xml',
+  'glapi/gen/ARB_sample_shading.xml',
   'glapi/gen/ARB_seamless_cube_map.xml',
   'glapi/gen/ARB_separate_shader_objects.xml',
   'glapi/gen/ARB_shader_atomic_counters.xml',
   'glapi/gen/ARB_shader_image_load_store.xml',
-  'glapi/gen/ARB_shader_subroutine.xml',
   'glapi/gen/ARB_shader_storage_buffer_object.xml',
+  'glapi/gen/ARB_shader_subroutine.xml',
+  'glapi/gen/ARB_shading_language_include.xml',
   'glapi/gen/ARB_sparse_buffer.xml',
   'glapi/gen/ARB_sparse_texture.xml',
   'glapi/gen/ARB_spirv_extensions.xml',
@@ -83,17 +82,15 @@ glapi_xml_py_deps = files(
   'glapi/gen/ARB_vertex_array_object.xml',
   'glapi/gen/ARB_vertex_attrib_64bit.xml',
   'glapi/gen/ARB_vertex_attrib_binding.xml',
-  'glapi/gen/ARB_viewport_array.xml',
-  'glapi/gen/AMD_depth_clamp_separate.xml',
-  'glapi/gen/AMD_draw_buffers_blend.xml',
-  'glapi/gen/AMD_gpu_shader_int64.xml',
-  'glapi/gen/AMD_performance_monitor.xml',
   'glapi/gen/ARB_vertex_type_2_10_10_10_rev.xml',
+  'glapi/gen/ARB_viewport_array.xml',
+  'glapi/gen/es_EXT.xml',
   'glapi/gen/EXT_direct_state_access.xml',
   'glapi/gen/EXT_draw_buffers2.xml',
-  'glapi/gen/EXT_external_objects.xml',
+  'glapi/gen/EXT_EGL_image_storage.xml',
   'glapi/gen/EXT_external_objects_fd.xml',
   'glapi/gen/EXT_external_objects_win32.xml',
+  'glapi/gen/EXT_external_objects.xml',
   'glapi/gen/EXT_framebuffer_object.xml',
   'glapi/gen/EXT_gpu_shader4.xml',
   'glapi/gen/EXT_multisampled_render_to_texture.xml',
@@ -103,33 +100,41 @@ glapi_xml_py_deps = files(
   'glapi/gen/EXT_shader_image_load_store.xml',
   'glapi/gen/EXT_texture_array.xml',
   'glapi/gen/EXT_texture_integer.xml',
-  'glapi/gen/EXT_texture_storage.xml',
   'glapi/gen/EXT_texture_storage_compression.xml',
+  'glapi/gen/EXT_texture_storage.xml',
   'glapi/gen/EXT_transform_feedback.xml',
   'glapi/gen/EXT_vertex_attrib_64bit.xml',
   'glapi/gen/EXT_window_rectangles.xml',
+  'glapi/gen/GL3x.xml',
+  'glapi/gen/GL4x.xml',
+  'glapi/gen/gl_and_es_API.xml',
+  'glapi/gen/gl_and_glX_API.xml',
+  'glapi/gen/gl_API.xml',
+  'glapi/gen/glX_API.xml',
   'glapi/gen/GREMEDY_string_marker.xml',
   'glapi/gen/INTEL_performance_query.xml',
-  'glapi/gen/KHR_debug.xml',
   'glapi/gen/KHR_context_flush_control.xml',
-  'glapi/gen/KHR_robustness.xml',
+  'glapi/gen/KHR_debug.xml',
   'glapi/gen/KHR_robustness_es.xml',
+  'glapi/gen/KHR_robustness.xml',
   'glapi/gen/KHR_shader_subgroup.xml',
   'glapi/gen/KHR_texture_compression_astc.xml',
+  'glapi/gen/MESA_tile_raster_order.xml',
   'glapi/gen/NV_alpha_to_coverage_dither_control.xml',
-  'glapi/gen/NV_copy_image.xml',
   'glapi/gen/NV_conditional_render.xml',
+  'glapi/gen/NV_copy_image.xml',
+  'glapi/gen/NV_half_float.xml',
   'glapi/gen/NV_primitive_restart.xml',
   'glapi/gen/NV_texture_barrier.xml',
+  'glapi/gen/NV_timeline_semaphore.xml',
   'glapi/gen/NV_vdpau_interop.xml',
   'glapi/gen/NV_viewport_swizzle.xml',
   'glapi/gen/OES_EGL_image.xml',
   'glapi/gen/OES_fixed_point.xml',
   'glapi/gen/OES_single_precision.xml',
   'glapi/gen/OES_texture_compression_astc.xml',
+  'glapi/gen/OVR_multiview_multisampled_render_to_texture.xml',
   'glapi/gen/OVR_multiview.xml',
-  'glapi/gen/GL3x.xml',
-  'glapi/gen/GL4x.xml',
   'glapi/registry/gl.xml',
 
   # Python files
